lib/engine/tile.rb

```diff
@@ -456,6 +456,10 @@ def label=(label_name)
       @label = Part::Label.new(label_name)
     end
 
+    def clear_label!
```

Comment:
remove this, and change the setter to handle nil

Reply:
Gotcha!